src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/AsyncHelpers.CoreCLR.cs

```diff
@@ -61,6 +61,15 @@ internal enum CorInfoContinuationFlags
         // OSR method saved in the beginning of 'Data', or -1 if the continuation
         // belongs to a tier 0 method.
         CORINFO_CONTINUATION_OSR_IL_OFFSET_IN_DATA = 4,
+        // If this bit is set the continuation should continue on the thread
+        // pool.
+        CORINFO_CONTINUATION_CONTINUE_ON_THREAD_POOL = 8,
+        // If this bit is set the continuation has a SynchronizationContext
+        // that we should continue on.
+        CORINFO_CONTINUATION_CONTINUE_ON_CAPTURED_SYNCHRONIZATION_CONTEXT = 16,
+        // If this bit is set the continuation has a TaskScheduler
+        // that we should continue on.
+        CORINFO_CONTINUATION_CONTINUE_ON_CAPTURED_TASK_SCHEDULER = 32,
```

Comment:
No continuation flags set means the continuation runs "inline", right?

Reply:
Right. That's the behavior for continuations created because of awaiting custom awaiters (via `UnsafeAwaitAwaiter`/`AwaitAwaiter`).